tools/autograd/derivatives.yaml

```diff
@@ -635,6 +635,9 @@
 - name: _standard_gamma_grad(Tensor self, Tensor output)
   self: not_implemented("_standard_gamma_grad")
 
+- name: _scalar_sum(Tensor self)
+  self: _scalar_sum_backward(grad, self)
```

Comment:
`grad.expand(self.sizes())`

Don't make people jump to another file to read such a simple gradient definition.

Reply:
hehe, this actually doesn't work and one of the reasons I disabled the test for being too fragile.  The reason is we change the size of things inplace (to get around the scalar blocks) so you actually need the tensor to get the right size, not the saved sizes of tensor.